src/components/Button/index.jsx

```diff
@@ -18,11 +18,12 @@ import {
  * Use instead of passing child text
  * @returns {JSX.Element}
  */
-export default function Button(props) {
+const Button = forwardRef((props, ref) => {
```

Comment:
I originally didn't have a reason to forward refs, but since I'm using it in `Popover.Button`'s `as` property, it was throwing an error since I'm assuming whatever it's doing behind the scenes requires the ref.